jobs/JRRFS_GEMPAK

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+date
+export PS4='+ $SECONDS + '
+set -xue
+
+
+#-----------------------------------------------------------------------
+# Source the variable definitions file
+#-----------------------------------------------------------------------
+#
+. ${HOMErrfs}/ush/rrfs_nco_config.sh
+
+#-----------------------------------------------------------------------
+# Specify Execution Areas
+#-----------------------------------------------------------------------
+
+export EXECrrfs=${EXECrrfs:-${HOMErrfs}/exec}
+export FIXrrfs=${FIXrrfs:-${HOMErrfs}/fix}
+export PARMrrfs=${PARMrrfs:-${HOMErrfs}/parm}
+export USHrrfs=${USHrrfs:-${HOMErrfs}/ush}
+export FIXam=${FIXam:-${FIXrrfs}/am}
+
+export RRFSFIXgem=${RRFSFIXgem:-${FIXrrfs/gempak/fix}
+
+
+#-----------------------------------------------------------------------
+# Source the bash utility functions.
+#-----------------------------------------------------------------------
+
+. $USHrrfs/source_util_funcs.sh
+
+export COMrrfs=${COMrrfs:-$(compath.py -o rrfs/${rrfs_ver})}
+export COMOUT=${COMOUT:-$(compath.py -o rrfs/${rrfs_ver}/${RUN}.${PDY}/${cyc}/gempak)}
+
+if [ ! -f ${COMOUT} ] ; then
+	mkdir -p -m 775 ${COMOUT}
+fi
+
+#################################################
+# Set up model and cycle specific variables
+#################################################
+
+export model=`echo $RUN | awk '{print tolower($0)}'`
+export GRIB=prslev
+export DBN_ALERT_TYPE=RRFS_GEMPAK
+
+if [ -e ./poescript ]
```

Comment:
Why is finding and delete the poescript before cd to DATA? The poe script should be located inside the DATA location.